lib/wallaby/session.ex

```diff
@@ -44,11 +44,12 @@ defmodule Wallaby.Session do
     id: String.t,
     session_url: String.t,
     url: String.t,
-    server: pid(),
-    screenshots: list
+    server: pid | nil,
+    screenshots: list,
+    driver: module
```

Comment:
I'm a big fan of this idea. This gives us a lot of flexibility for running different browsers against each other.